### ISSUE TITLE
Reviewer 1 response

### ISSUE THREAD

**Issue by PaulSpence** [adele157/easterlies-collaborative-project#74]
Reviewer 1: Lines 103-115: I am aware of Kiss et al., 2020. However, I still recommend
authors show (1) the difference in their CONTROL compared to Kiss et al., 2020 and (2)
model evaluation compared to observations. I recommend showing simulated T and S for
coastal regions, comparison of sea ice extent, sea ice formation rate, AABW production, etc.
Even if it is similar to Kiss et al., model evaluation is crucial for readers to understand the
meanings of sensitivity experiments. If authors use the same run compared to Kiss et al., I
would like authors to repeat this information for new readers and cite papers including figure
numbers.

Reply: Paul: We use RYF which isn’t in Kiss et al. Is there another paper that evaluates
the RYF in this region that we can cite, e.g. Stewart et al? If not, I suggest: Add an
observational temp, salt and rho contour lines to Fig. 5a-c. Add an obs contour line to Fig.
5d. What obs T/S do we use? Just the model initial conditions? Also add sea ice ext control
and obs to Fig. 7

**Comment by PaulSpence** [comment]
I redid Fig 5 trying to overlay a 27.75 contour from the model initial conditions. It doesn't look very good. Below are the initial conditions temp and model temp with the 27.75 isopycnals (green is the initial condition, black the model).
![Screen Shot 2023-03-22 at 12 51 57 pm](https://user-images.githubusercontent.com/6644956/226781581-5f647870-9da2-452f-8ab8-6d71ddbac45e.png)

**Comment by PaulSpence** [comment]
Below is the initial condition and model rho. The init conditions has contour lines at 27.75, 27.65 and 27.55.
![Screen Shot 2023-03-22 at 12 54 34 pm](https://user-images.githubusercontent.com/6644956/226781838-84591d0e-a391-4e13-91db-d1f8ae96ba42.png)

**Comment by PaulSpence** [comment]
Salt comparison:
![Screen Shot 2023-03-22 at 12 56 22 pm](https://user-images.githubusercontent.com/6644956/226781984-bbfeefd9-e4af-4b6e-817f-483b0de246d4.png)

**Comment by adele157** [comment]
Hmmm, yeah it looks a bit messy. What about just putting maps of T/S bias in the supplementary?

**Comment by PaulSpence** [comment]
Yep. I agree. Looks like crap :). Stash in supplementary. I am curious to see how 5d looks ... e.g. is CDW outside of upwelling range in obs?

**Comment by PaulSpence** [comment]
Re: Fig 7 sea ice evaluation relative to obs. Below the black contour is annual mean sea ice extent (conc=0.15) in the RYF control and the 1985-1995 mean from NOAA/G02202_V3 obs data. We would need to extend the map to ~55S to show the full contour - at the expense of clarity of the thickness and vectors. Also, others may want to see extent change in the perturbation. Thoughts?

![Screen Shot 2023-03-22 at 4 03 54 pm](https://user-images.githubusercontent.com/6644956/226807215-02bcf690-afe3-4a5d-aa6f-30cba42c1081.png)

**Comment by adele157** [comment]
Yeah, I think my vote is for keeping these figures clean and putting the validation in the supplementary again?

**Comment by PaulSpence** [comment]
Cool. I'll create all these obs/model comparisons as supplementary figs. Hows this for the sea ice extent?
![Screen Shot 2023-03-22 at 5 49 44 pm](https://user-images.githubusercontent.com/6644956/226824239-07c1b172-8527-4653-b308-8bcb19571ce8.png)

**Comment by adele157** [comment]
@PaulSpence for the WOA comparison plots above, sorry to be picky, but it's maybe not so good to compare with model initial conditions, because these use January data in the upper ocean, so will be biased compared with a model annual average.

There are monthly averages of WOA data on the model grid here:
`/g/data/ik11/observations/woa13/01/` that we could use. If you don't have time, you could upload your scripts to github (probably good to upload anyway) and I can modify.

**Comment by adele157** [comment]
Also for sea ice, what about adding maps of concentration at low and high months, because extent doesn't say much about what the coastal coverage is like. e.g. could redo this figure from Kiss et al. 2020:
<img width="471" alt="Screen Shot 2023-03-30 at 4 54 45 pm" src="https://user-images.githubusercontent.com/8506963/228742606-ef5e5595-1e60-440c-8857-0d00244718a3.png">

**Comment by adele157** [comment]
Also the bottom T/S from WOA looks really wacky. Why is it so cold in the southern Amundsen / Bellingshausen? And Totten and Vincennes Bay are ridiculously warm!

<img width="635" alt="Screen Shot 2023-03-30 at 5 10 14 pm" src="https://user-images.githubusercontent.com/8506963/228745230-76b6f9fd-4173-4928-93fd-b9325f9cd25a.png">

For comparison, here is Schmidtko:
<img width="812" alt="Screen Shot 2023-03-30 at 5 10 34 pm" src="https://user-images.githubusercontent.com/8506963/228745533-f441d317-263c-44ed-9fec-c9744fea3435.png">

I reckon we should compare to Schmidtko instead. There is [code here](https://github.com/pedrocol/basal_mom5-collaborative-project/blob/main/notebooks/comp_obs/temp_salt_time_series.ipynb) for loading and plotting Schmidtko data.

**Comment by wghuneke** [comment]
Downside of the Schmidtko data is that there're none in East Antarctica (~ 90-130E)

**Comment by adele157** [comment]
Probably that means there’s not enough bottom data there for it to be worth
a comparison then? The Pauthenet climatology only goes down to 400m for
similar reasons.

We are also comparing to WOA on the 1000m isobath, so still capture some of
the bottom outflow properties compared to WOA obs. The WOA just looks crazy
to me on the shelf.
